### PR TITLE
Tidy platform readme

### PR DESCRIPTION
**Pull request by rafahop**
### What

Minor changes in readme

### CODE REVIEW

File: training/platform-services/PLATFORM.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Platform Services
 ===========================
 
-Platform services allow us to develop, run, and manage applications without the complexity of building and maintaining the infrastructure typically associated with developing and launching an app. This documentation gives information on the platform services that we use to assist in deploying and monitoring applications.
+Platform services allow us to develop, run and manage applications without the complexity of building and maintaining the infrastructure typically associated with developing and launching an app. This documentation gives information on the platform services that we use to assist in deploying and monitoring applications.
```
Comment (gedge) [inline training/platform-services/PLATFORM.md:4]: not a fan of the Oxford comma, eh? me neither, but it's not _wrong_

File: training/platform-services/PLATFORM.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ Visit https://concourse.onsdigital.co.uk to access Concourse
 [<img src="images/concourse-home.png" title="Concourse Homepage" width=49.7% height=49.7%/>][concourse-ons] [<img src="images/concourse-pipeline.png" title="Concourse Pipeline" width=49.7% height=49.7%/>][concourse-ons]
 
 Concourse lists all of the applications where each application has its own [pipeline][concourse-pipeline]. 
-When a [PR][git-pr] of a application is said to be merged into `develop` or `master`, Concourse runs the pipeline of the application which does unit testing, checks if a [docker image][docker-image] can be build for it, bundles it for release and ships it to that release whilst ensuring that it has deployed everything completely.
+When a [PR][git-pr] of an application is said to be merged into `develop` or `master/main`, Concourse runs the pipeline of the application which does unit testing, checks if a [docker image][docker-image] can be built for it, bundles it for release and ships it to that release whilst ensuring that it has deployed everything completely.
```
Comment (gedge) [inline training/platform-services/PLATFORM.md:43]:
```suggestion
When a [PR][git-pr] of an application is said to be merged into `develop` or `master`/`main`, Concourse runs the pipeline of the application which does unit testing, checks if a [docker image][docker-image] can be built for it, bundles it for release and ships it to that release whilst ensuring that it has deployed everything completely.
```

Comment (gedge) [reply training/platform-services/PLATFORM.md:43]: `master/main` is a valid branch name

Comment (rafahop) [reply training/platform-services/PLATFORM.md:43]: Ha good point!

**Review by gedge — APPROVED** [review]

**Pull request merged**